packages/amplify-provider-awscloudformation/src/download-api-models.ts

```diff
@@ -50,6 +50,9 @@ const extractAPIModel = async (context: $TSContext, resource: $TSObject, framewo
 
   fs.ensureDirSync(tempDir);
 
+  // This has been working fine before bumping node types to 18.x
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I had to suppress compiler here after bumping node typings...
The alternative would be 5 logical branches to handle union type of `data.body` .

Since this was working fine, I suggest we fix this later as a follow up.

Reply:
Could you add the comment in the PR to the code? (same for zip-util)

Reading the commend alone in the code doesn't make much sense.

Reply:
done.